src/metabase/actions.clj

```diff
@@ -199,25 +199,6 @@
    :actions.args/common
    (s/keys :req-un [:actions.args.crud.row.common/query])))
 
-;;; the various `:row/*` Actions all treat their args map as an MBQL query.
-
-(defn- normalize-as-mbql-query
```

Comment:
I also removed the custom actions normalization code and rolled it in to the normal MBQL normalization code